src/main/resources/application-prod.yml

```
@@ -12,8 +12,30 @@ spring:
+  level:
+    org:
+      hibernate:
+        sql: warn
+        type.descriptor.sql: trace
+jwt:
+  secret: ${JWT_SECRET}
+  access-expiration: ${JWT_ACCESS_EXPIRATION:3600000}
+  refresh-expiration: ${JWT_REFRESH_EXPIRATION:604800000}
+file:
+  profile-image-dir: C:/Users/user/profile-images
+  profile-image-url-base: /uploads/profiles/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)